src/components/ui/ClusterEventsList.test.tsx

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import ClusterEventsList, { ClusterEventsListProps } from './ClusterEventsList';
+
+// TODO: this fixture needs a warm place where to live in the project.
+const props = Object.freeze<ClusterEventsListProps>({
```

Comment:
Since this test has already been written, let's keep it here. We can either maintain or remove later.
Anyway, I would prefer to rewrite that test to the Cypress in a follow-up.